openedx/core/djangoapps/programs/models.py

```diff
@@ -132,3 +137,10 @@ def is_certification_enabled(self):
         certificates for Program completion.
         """
         return self.enabled and self.enable_certification
+
+    @property
+    def show_xseries_ad(self):
```

Comment:
I'm concerned about how tightly tied this property and the underlying field are to the XSeries product. We should strive to keep this more generic. For example, the field could be called `find_more_programs`. A separate field on this model can hold the value to be used when constructing the link; right now, it's hardcoded as 'xseries'.